vaadin-date-picker-flow-parent/vaadin-date-picker-flow/date-fns/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "vaadin-date-picker-date-fns",
+  "description": "Custom build of date-fns used by the Vaadin Flow Date Picker",
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
+  },
+  "keywords": [],
+  "author": "Vaadin Ltd",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "date-fns": "^2.23.0"
```

Comment:
versions should be fixed to provide reproducible builds

Reply:
Done